DataBiosphere/leonardo
# Async Cluster Creation - Phase 1

I'm opening this prematurely to get feedback while I look

1. test with dev db 
2. address automation failures, if any

and if time permits,
2. see whether `getByGoogleId` and `getById` can be removed
2. remove `assertEquivalent` in favour of `ClusterEnrichments.clusterEq` for `Cluster` equivalency checks

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:


In all cases:

- [ ] Squash and merge; you can delete your branch after this
